src/pages/2x-grid.mdx

```diff
@@ -335,16 +335,21 @@ The 8px mini unit guides everything in both software product and web environment
       actionIcon="arrowRight"
       >
 
-![](/images/bee.svg)
+![](/images/carbon.png)
 
   </ResourceCard>
 </Column>
 <Column colMd={4} colLg={4} noGutterSm>
-    <ResourceCard
-      subTitle="IBM digital media templates"
+      <ResourceCard
+      subTitle="2x Grid in UI: Implementation"
       aspectRatio="2:1"
-      disabled
-      ></ResourceCard>
+      href="https://www.carbondesignsystem.com/guidelines/2x-grid/implementation"
+      actionIcon="arrowRight"
```

Comment:
we can remove the `actionIcon` here since we want to use the default `"launch"` icon. Same for the resource card on the left. Both of them should have the `"launch"` icon, so we can remove the prop on that one too (line 335).